tar/multitape/multitape_delete.c

```diff
@@ -76,6 +77,7 @@ deletetape(TAPE_D * d, uint64_t machinenum, const char * cachedir,
 	int lockfd;
 	uint8_t lastseq[32];
 	uint8_t seqnum[32];
+	int rc = -1;	/* Assume an error will occur; will change if needed. */
```

Comment:
We will be doing `return (rc)`, so we want `rc` to be the right value.  This value can be 0, 1, or -1.  If we set it to -1 in err3, then we won't be returning 1 when we want to.  So instead, the plan is to set `rc=-1` by default.  If we hit the FILENOTFOUND, then we set `rc=1`.  If we reach `/* Success */` then we set `rc=0`.

I mean, I *could* add a separate `int did_we_receive_filenotfound` and in `err0` set rc as either -1 or 1, but that seemed silly.  I'm confident about my logic here, but not about the comment to explain it.